src/components/marketingDialog/theme.css

```diff
@@ -1,8 +1,8 @@
 .marketing-dialog {
-  width: 900px;
+  width: 1000px;
```

Comment:
Q: Can we use the `hero` width (985px) instead of the override or is it still too small?

Reply:
Yes it fits :+1: